### PR TITLE
[MAINTAIN-12] Small Banner paragraph should be responsive in Rose theme

### PR DESCRIPTION
**Pull request by aleevas**
Original Issue, this PR is going to fix: [MAINTAIN-12](https://openy.atlassian.net/browse/MAINTAIN-12)

This one should add ability to Small Banner paragraph to be responsive with big amount of text.
Now if you want to add a much more text than banner height can store -> the font-size in the banner will should be decreased

MAINTAIN-12SmallBannerWithBigTextRose
MAINTAIN-12SmallBannerWithBigTextRoseTablet
MAINTAIN-12SmallBannerWithBigTextRoseMobile

## Steps for review

- [ ] Log as admin
- [ ] go to edit home page
- [ ] add a new Small Banner block with long title and description
- [ ] verify that you can see all text in this banner
- [ ] verify on different type of screens

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3135/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3135/

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3135/

**Comment by podarok** [comment]
Looks good
image

**Review by podarok — APPROVED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3136/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3136/

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3136/

File: themes/openy_themes/openy_rose/scripts/openy_rose.js [hunk 1]
```diff
@@ -274,4 +274,30 @@
...
+      const headlineMaxLength = 45
+      const  descriptionMaxLength = 325
+
+      let headline = $('.banner--small .field-prgf-headline'),
+          description =  $('.banner--small .field-prgf-description p')
+
+      // Decrease font size if content is too long.
+      if (headline.text().length > headlineMaxLength) {
+        headline.css({
+          fontSize: '22px',
```
Comment (hamrant) [inline themes/openy_themes/openy_rose/scripts/openy_rose.js]: Let's try to use relative sizes, for example, `vw` - https://css-tricks.com/almanac/properties/f/font-size/

**Review by hamrant — CHANGES_REQUESTED** [review]
On some screen sizes text not visible:

Screenshot from 2021-04-12 19-24-33

**Comment by hamrant** [comment]
@aleevas anyway, now it looks better:

Screenshot from 2021-04-12 19-19-16

Screenshot from 2021-04-12 19-24-12

**Comment by gundevel** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3137/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3137/

**Comment by gundevel** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3137/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3138/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3138/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3138/

**Review by hamrant — APPROVED** [review]
Looks ok to me

Screenshot from 2021-04-13 15-50-29

Screenshot from 2021-04-13 15-51-08

Screenshot from 2021-04-13 15-50-46

On window resize text can be too small, but I think this not critical

**Review by podarok — APPROVED** [review]

**Pull request merged**